tf5muxserver/mux_server_GetMetadata.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // GetMetadata merges the metadata returned by the
 // tfprotov5.ProviderServers associated with muxServer into a single response.
-// Resources, data sources, ephemeral resources, and functions must be returned
+// Resources, data sources, ephemeral resources, list resources, and functions must be returned
 // from only one server or an error diagnostic is returned.
 func (s *muxServer) GetMetadata(ctx context.Context, req *tfprotov5.GetMetadataRequest) (*tfprotov5.GetMetadataResponse, error) {
```

Comment:
Shall we also update the test in `tf5muxserver/mux_server_GetMetadata_test.go`? & similarly for v6.